src/transformers/models/layoutlmv3/configuration_layoutlmv3.py

```diff
@@ -203,7 +203,7 @@ def inputs(self) -> Mapping[str, Mapping[int, str]]:
                     ("input_ids", {0: "batch", 1: "sequence"}),
                     ("attention_mask", {0: "batch", 1: "sequence"}),
                     ("bbox", {0: "batch", 1: "sequence"}),
-                    ("pixel_values", {0: "batch", 1: "sequence"}),
+                    ("pixel_values", {0: "batch", 1: "num_channels", 2: "height", 3: "width"}),
```

Comment:
Following https://github.com/huggingface/transformers/pull/17976 I've renamed `sequence` to `num_channels`